README.md

```diff
@@ -1,18 +1,18 @@
+0 | Denial | <ul><li>The Product/Service project group is homogenous and works in silos.</ul></li><ul><li>Decisions are centred top-down instead of the "Circle Way".</ul></li><ul><li>More time is spent planning and building without a complex feedback system or feedback prioritized.</ul></li><ul><li>Making for an outdated product/service prior to release</ul></li><ul><li>"Firefighting" approach is normalized, compromising long-term (ie. generational) planning development.</ul></li></ul><ul><li>Data is not clean nor secure, demograpic data (race, gender and immigration) isn't relevant in decision making and decisions are based on assimilation and/or capital, leading to hypervisibility or invisibility of marginalized groups.</ul></li><ul><li> Numerous privacy breaches, deemed to harm civil liberities, have been reported.</ul></li></ul></li> <ul><li>Metrics are valued more than customer experience - basic language script is utilized and accessibility is non-existant (automated speech recognition  systems, triggering language, not optimized for Braille, etc.)</ul></li> <ul><li> Heavy documentation is common but record management policies are either ignored or unknown.</ul></li><ul><li> It has not been tested with various minority groups (BIPOC, LGBTQIA2S, Gender Diverse, People with Disabilities, Neuro-Diverse, Residents of Rual Areas).</ul></li>
+1 | Polarization | <ul><li>Project group accommodates and/or inclusive of some miniorities rather than all representatives, mainly for optical purposes.</ul></li><ul><li>Volume of work is prioritized from efforts for efficiency and organization.</ul></li><ul><li>Feedback process follows antiquated methodology, is irregular, non-existant or biased, only being initiated if a large problem is brought forward or a large investment is involved; feedback and areas of improvement from minorities are either minimalized, avoided, and/or ignored.</ul></li><ul><li>"Walking on Thin Ice" approach is common as potentially sensitive project elements, such as accessibility and demographic data, is regarded with fear of open conflict - inequalities and large data gaps are pravalent.</ul></li><ul><li>Product/Service data are collected from random sample(s) of users instead of users who maintain, use and implement the product/service.</ul></li><ul><li>Records management is acknowledged and approached with simplicity rather than complexity such as consideration of electronic EDRMS folders, off-site records maintenance, different types of storage, etc.</ul></li>
```

Comment:
Project group accommodates and/or inclusive of some miniorities rather than all representatives, mainly for optical purposes.
=> Project group accommodates some rather than all representatives; favours appearances over outcomes.

Reply:
Volume of work is prioritized from efforts for efficiency and organization.
=> Volume of work is prioritized over efficiency and organization.

Reply:
Completed.